ksql-engine/src/test/java/TestUdfWithNoPackage.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+
+import io.confluent.ksql.function.UdfCompiler;
+import io.confluent.ksql.function.UdfInvoker;
+import java.util.Optional;
+import org.junit.Test;
+
+public class TestUdfWithNoPackage {
```

Comment:
Why not add this to the `UdfCompilerTest.java` instead of a new class?

Reply:
I can't 😂 can't import the udf from a class that has no package

Reply:
Maybe add a java doc to this class to avoid others being confused and maybe even moving this class!

Might even be worth adding a meta-test in here that fails if this class has a package.

Reply:
For some reason, when running this via Jenkins my test for package returns `null` instead of `""` (what it returns locally!)

Reply:
Evil! JDK 9 had a change that JDK 8 did not have...